packages/swapper/src/swappers/JupiterSwapper/errors.ts

```diff
@@ -0,0 +1,23 @@
+import type { SwapperErrorMapping } from '../baseErrors'
+import { SWAPPER_ERRORS } from '../baseErrors'
+
+export const JUPITER_ERROR_VALUES = {
+  SLIPPAGE_EXCEEDED: 'SlippageToleranceExceeded',
```

Comment:
q: do we even feel like errors at execution time is something that generalizes to the point we shouls abstract this away? Feels like all we're doing here could be done wherever the error bubbles up (`useTradeExecution()`) 

From what I'm seeing, this is really solana/jupiter specific, and none of our other supported chains would return errors at Tx broadcast time, are you aware of other Solana swappers doing the same as jupiter @NeOMakinG ?

Reply:
The way we've done it is now throwing specific error types from the swapper easing the handling and seems like the right home, it's also a good pattern we could take in the future, wdyt?

Reply:
Assuming we want to go from broadcast error to generalized error monad, 100%!